report_partner_address/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Report Partner Address",
```

Comment:
```suggestion
    "name": "Report Partner Address",
    "summary": "Translatable partner address details for reports and portal",
```

I just realized that the summary affects the module presentation in Odoo's app store (it shows when you hover over the module). So it would be a good idea to always include it.